TEMPLATE.md

```diff
@@ -0,0 +1,87 @@
+# Tech Proposal Template
+
+
+
+This template is for Haskell Foundation technical agenda proposals. You may find existing proposals and more examples at the [tech-proposals](https://github.com/haskellfoundation/tech-proposals) repository. 
+
+
+
+## Introduction
+
+This section should detail the following: A summary of the proposal’s
+motivation, implementation, risks, timeline, and goals. It should aim to
+be an abstract for the proposals.
+
+## Background
+
+This section should explain any background (targeting a casual audience)
+needed to understand the proposal’s motivation (e.g. a high level
+overview of the technical details and some history).
+
+
+
+## Motivation
+
+This section should explain the motivation in terms of why Haskell
+Foundation should take on this project, and why it is important for the
+Haskell ecosystem. What impact will it have? Why should people care?
+
+## Goals
+
+This section should detail the high-level goals of the project, as well
+as the criteria for what constitutes a finished project. Deliverables
+should support these goals, and the goals should support the motivation
+for undertaking the project.
+
+## People
+
+This section should detail the following people:
+
+-   **Performers:** the project leads and main developers/resources who will be directly involved in the day to day work.
```

Comment:
```suggestion
-   **Performers:** the project leads and main developers/resources who will be directly involved in the day to day work. If your project needs performers to be sourced by the Haskell Foundation, please say so here.
```

I would want the possibility of our finding volunteers for a worthwhile project to be explicitly possible. Of course, a proposal with performers attached is more compelling than one without.